### PR TITLE
Utilities for measuring time and for avoiding stack traces

### PR DESCRIPTION
**Pull request by mjambon**

### CODE REVIEW

**Review by aryx — APPROVED** [review]

File: h_program-lang/Error_code.ml [hunk 1]
```diff
@@ -405,6 +405,15 @@ let try_with_print_exn_and_reraise file f =
     (* does not really re-raise :( lose some backtrace *)
     raise exn
 
+(* fast = no stack trace *)
+let try_with_print_exn_and_exit_fast file f =
+  try
+    f ()
+  with exn ->
```
Comment (IagoAbal) [inline h_program-lang/Error_code.ml:412]: If you don't need the stack trace for some exception you can also use `raise_notrace`.

Comment (mjambon) [reply h_program-lang/Error_code.ml:412]: What will `get_backtrace ()` do then when it gets called? The manual says nothing, about it, I'm filing a ticket.

Comment (IagoAbal) [reply h_program-lang/Error_code.ml:412]: I guess the empty string, but it would be great if it was specified!

Comment (mjambon) [reply h_program-lang/Error_code.ml:412]: https://github.com/ocaml/ocaml/issues/10467

**Pull request merged**